imagetagger/imagetagger/annotations/static/annotations/js/annotations.js

```diff
@@ -932,6 +939,20 @@ function calculateImageScale() {
     } else {
       handleNewAnnotations();
     }
+
+  }
+
+  function loadAnnotated() {
+    let allKeys = Object.keys(gAnnotationCache);
+    for(let i = 0; i < allKeys.length; i++) {
```

Comment:
I think you can use a [for...in loop](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in) rather than getting the keys then looping over the indices in the array. Can probably remove `allKeys` as a result.